tests/test_history.py

```diff
@@ -68,5 +71,75 @@ def test_cmd_field():
     yield assert_equal, None, hist.outs[-1]
     os.remove(FNAME)
 
+def test_show_cmd():
+    FNAME = 'xonsh-SESSIONID.json'
+    FNAME += '.show_cmd'
+    cmds = ['ls', 'cat hello kitty', 'abc', 'def', 'touch me', 'grep from me']
+
+    def FormatHistLine(idx, cmd):
```

Comment:
Oops. Sorry about that. My employer uses a modified PEP8 style where function names are in camel-case. Fixed.